### ISSUE TITLE
Error when executing a sys.ls

### ISSUE THREAD

**Issue by will-chan** [gptscript-ai/desktop#82]
I added a file into my workspace and asked the default assistant "what is in my workspace".  It errors up with:

<img width="670" alt="Screenshot 2024-08-12 at 2 25 32 PM" src="https://github.com/user-attachments/assets/2b4de0c5-80d5-4436-a0e4-f53e85bb8cf4">

**Comment by thedadams** [comment]
OK. I think I have figured out the issue here, and it's very strange.

The LLM doesn't explicitly have the sys.ls tool but it intuits the existence of the tool from the prompt in the github.com/gptscript-ai/tildy tool.

When LLM tries to use it, it will request that the sys.ls function be called. GPTScript will do that and send the response back to the LLM. However, OpenAI doesn't like the . in the function name and blows up.

If you give the tildy tool explicit access to the sys.ls tool, then gptscript will add it as ls and the LLM uses it properly.

TL;DR: this is a problem with the tildy tool.

**Comment by thedadams** [comment]
@sangee2004 I believe this has been addressed.

**Comment by sangee2004** [comment]
This issue is not seen anymore when testing with latest version of  tildy tool.
